### ISSUE TITLE
layout: implement minimalistic renderer

### ISSUE THREAD

**Issue by arekkas** [ory/editor#250]
* [x] have minimalistic bundle/component that renders the content without the editor, but with plugins
* [x] sanitize html output of said component

**Comment by arekkas** [comment]
I think there are multiple things we want:

1. Keep the bundle small: We don't want to include all the third-party stuff that we use in edit mode: material-ui, react-dnd, and so on.
2. Keep the mark-up clean
3. Render server-side

Thus I think it would make sense to have an extra component that executes the row/cell and plugin logic only, resulting in minimal mark up:

```
<div class="row">
  <div class="cell-9">
    <div class="some-plugin />
  </div>
</div>
```

Then, this component could easily be used in a production app:

```jsx
import Render from 'ory-editor/src/components/Renderer

ReactDOM.renderToMarkup(<Renderer state={state} />)
```

What do you think ping @kevgo @inyono ?

**Comment by arekkas** [comment]
resolved see #324
